lib/publiccloud/provider.pm

```diff
@@ -196,9 +196,10 @@ Creates ~/.ssh/config file with all the common ssh client settings
 
 sub place_ssh_config {
     # configure ssh client
-    my $ssh_config_url = data_url('publiccloud/ssh_config');
+    my $ssh_config_url = data_url(get_var('PUBLIC_CLOUD_SSH_CONFIG', 'publiccloud/ssh_config'));
     assert_script_run("curl $ssh_config_url -o ~/.ssh/config");
     file_content_replace("~/.ssh/config", "%SSH_KEY%" => get_ssh_private_key_path());
+    assert_script_run("cat ~/.ssh/config");
```

Comment:
is for debug only? Do you plan to keep it in the final PR version? How do you see the idea to upload it to openQA instead of have it printed in the `mare magno` of setial_terminal.txt ?

Reply:
Removed.